rego/lib/fugue/regula/tests.rego

```diff
@@ -0,0 +1,8 @@
+package fugue.regula.tests
+
+import data.fugue.resource_view.resource_view_input
+
+mock_input(iac_configs) = ret {
+  iac_config = iac_configs[_]
+  ret = resource_view_input with input as iac_config.content
```

Comment:
Hmm, if `iac_configs` has more than one element this function will fail with an `inconsistent result` error.

Reply:
Thanks! I pushed a fix in 4b0dad5